src/Runner.Listener/JobDispatcher.cs

```diff
@@ -335,6 +344,12 @@ private async Task RunAsync(Pipelines.AgentJobRequestMessage message, string orc
             Busy = true;
             try
             {
+                //invoke JobStarted Event
+                if (JobStarted != null)
+                {
+                    JobStarted(this, new EventArgs());
```

Comment:
do we want to feed in some job info for tracing? ex jobId/name etc.

Reply:
I think it won't be necessary.